### ISSUE TITLE
Training EMAGE model with BEAT dataset

### ISSUE THREAD

**Issue by vothebao** [PantoMatrix/PantoMatrix#154]
Thank you so much for sharing this work!
I have a question: can we train the EMAGE model with the skeleton and ARKit blendshapes (like BEAT dataset)?

Thank you!

**Comment by fcchit** [comment]
I think BEAT-X can be seen as a subset of BEAT.

**Comment by H-Liu1997** [comment]
yes, it is possible, you could just replace the dataloader of beat2.py into BEAT format! thanks for your kind words!
